packages/arb-token-bridge-ui/src/util/deposits/fetchDeposits.ts

```diff
@@ -21,10 +17,7 @@ export type FetchDepositParams = {
   searchString?: string
 }
 
-/* Fetch complete deposits - both ETH and Token deposits from subgraph into one list */
-/* Also fills in any additional data required per transaction for our UI logic to work well */
-/* TODO : Add event logs as well */
-export const fetchDeposits = async ({
+export async function fetchDepositList({
```

Comment:
For consistency, we should rename the file to `fetchDepositList` now? Same comment for `fetchWithdrawalList`.

Reply:
Renamed